### PR TITLE
Google esp integration examples

### PR DESCRIPTION
**Pull request by jingyi-gao-ttd**

### CODE REVIEW

File: examples/google-esp-integration/with_sdk_v3/README.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
 # UID2 SDK ESP Integration Example
 
-This example demonstrates how a content publisher who are working with (Google Publisher Tags) can use the ESP (Google Encrypted Signals for Publishers) and the [Client-Side Identity JavaScript SDK](https://github.com/UnifiedID2/uid2docs/blob/main/api/v1/sdks/client-side-identity-v1.md) (also known as the UID2 SDK) to share UID2 direclty with bidders.
+This example demonstrates how a content publisher who is working with Google publisher tags can use the [Google Encrypted Signals program (ESP)](https://support.google.com/admanager/answer/10488752) and the [Client-Side Identity JavaScript SDK](https://github.com/UnifiedID2/uid2docs/blob/main/api/v1/sdks/client-side-identity-v1.md) (also known as the UID2 SDK) to share UID2 directly with bidders, in an implementation that uses this SDK.
```
Comment (jingyi-gao-ttd) [inline examples/google-esp-integration/with_sdk_v3/README.md:3]: Sorry I forgot to update Google Encrypted Signals program (ESP) to Google secure signals

File: examples/google-esp-integration/server_only/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Server-Only UID2 ESP Integration Example
+
+This example demonstrates how a content publisher who is working with [Google Interactive Media Ads(IMA) SDKs](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side) can use [Google Secure Signal](https://support.google.com/admanager/answer/10488752) to share UID2 directly with bidders, in a server-only implementation of UID2.
+
+For an example application using the [Client-Side Identity JavaScript SDK](https://github.com/IABTechLab/uid2docs/blob/main/api/v2/sdks/client-side-identity.md), see [UID2 SDK Integration Example](../with_sdk_v3/README.md).
```
Comment (jingyi-gao-ttd) [inline examples/google-esp-integration/server_only/README.md:5]: We need to update this to SDK v3

File: examples/google-esp-integration/with_sdk_v3/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# UID2 SDK ESP Integration Example
+
+This example demonstrates how a content publisher who is working with [Google Interactive Media Ads(IMA) SDKs](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side) can use [Google Secure Signal](https://support.google.com/admanager/answer/10488752) and the [Client-Side Identity JavaScript SDK](https://github.com/IABTechLab/uid2docs/blob/main/api/v2/sdks/client-side-identity.md) (also known as the UID2 SDK) to share UID2 directly with bidders, in an implementation that uses this SDK.
```
Comment (jingyi-gao-ttd) [inline examples/google-esp-integration/with_sdk_v3/README.md:3]: We need to update this to SDK v3

File: .github/workflows/docker-image.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
+name: Docker Image CI
+
+on:
+  workflow_dispatch:
+  push:
+# uncomment the above line to test this action on every push
```
Comment (alex-yau-ttd) [inline .github/workflows/docker-image.yml:6]: is this comment still accurate?

File: examples/google-esp-integration/server_only/public/ads.js [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+    // content has not actually ended.
+    if (isAdPlaying) return;
+    isContentFinished = true;
+    adsLoader.contentComplete();
+  };
+  videoContent.onended = contentEndedListener;
+
+  // Request video ads.
+  const adsRequest = new google.ima.AdsRequest();
+  adsRequest.adTagUrl = "${AD_TAG_URL}";
```
Comment (alex-yau-ttd) [inline examples/google-esp-integration/server_only/public/ads.js:59]: for future reference, this is the only non-formatting difference compared to google's code: https://github.com/googleads/googleads-ima-html5/blob/main/simple/ads.js

File: examples/google-esp-integration/with_sdk_v3/public/ads.js [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+    // content has not actually ended.
+    if (isAdPlaying) return;
+    isContentFinished = true;
+    adsLoader.contentComplete();
+  };
+  videoContent.onended = contentEndedListener;
+
+  // Request video ads.
+  const adsRequest = new google.ima.AdsRequest();
+  adsRequest.adTagUrl = "${AD_TAG_URL}";
```
Comment (alex-yau-ttd) [inline examples/google-esp-integration/with_sdk_v3/public/ads.js:59]: for future reference, this is the only non-formatting difference compared to google's code: https://github.com/googleads/googleads-ima-html5/blob/main/simple/ads.js

File: examples/google-esp-integration/with_sdk_v3/public/stylesheets/app.css [hunk 7]
```diff
@@ -0,0 +1,106 @@
+body {
```
Comment (alex-yau-ttd) [inline examples/google-esp-integration/with_sdk_v3/public/stylesheets/app.css:1]: I know these css files were copied from the example-sites repo - but is it worth addressing some css nits in this PR? There are some formatting issues, some missing semicolons, some `0px` and some unnecessary decimals e.g. `1.0px`. Autoformatter in vscode will fix most of them automatically.

Comment (alex-yau-ttd) [reply examples/google-esp-integration/with_sdk_v3/public/stylesheets/app.css:1]: same question applies to the server_only `app.css` too

File: webpack.config.js [hunk 8]
```diff
@@ -6,16 +6,7 @@ const espOutput = {
 };
 
 // n.b. if you add more outputs, the path is relative to the dist folder.
-const getExampleOutputs = (env) =>
-  !env.outputToExamples
-    ? {}
-    : {
-        ...espOutput,
-        mockedGoogleTag: {
```
Comment (alex-yau-ttd) [inline webpack.config.js:14]: we don't need this mock any more because we are getting a real google tag?

**Review by alex-yau-ttd — APPROVED** [review]

**Pull request merged**